### DISCUSSION TITLE
RollingFileAppender and tailing log file

### DISCUSSION THREAD

**Discussion by jmayday** [DataDog/datadog-agent#19445] [General]
I want to avoid my application log takes all available container space, so I configured rolling appender (Logback's RollingFileAppender) effectively limiting log size(s). Will Datadog always pick everything from log file app.log before it's being "archived" to app.1.log?

I can't find answer to this in DD documentation (https://docs.datadoghq.com/agent/logs/?tab=tailfiles).
Datadog agent is tailing from app.log file (path to file is configured in myapp.d/conf.yaml). Is this possible, that Java app (specifically rolling appender) will already move contents from main log file app.log, to app.1.log while Datadog agent didn't manage to read some of it's contents? May I end up having some log statements not picked up in such situation?

**Comment by saghayar** [comment]
Have a look at this link from datadog docs: 

https://docs.datadoghq.com/agent/logs/advanced_log_collection/?tab=configurationfile#tail-directories-using-wildcards
